odd-platform-ui/src/components/shared/AppTabs/AppTabs.tsx

```diff
@@ -36,15 +36,15 @@ const AppTabs: React.FC<AppTabsProps> = ({
 }) => {
   const [currentTab, setCurrent] = React.useState<
     number | boolean | undefined
-  >(selectedTab);
+  >(selectedTab === -1 ? false : selectedTab);
 
   const handleChange = (event: SyntheticEvent, newTab: number) => {
     setCurrent(newTab);
     handleTabChange(newTab);
   };
 
   React.useEffect(() => {
-    setCurrent(selectedTab);
+    setCurrent(selectedTab === -1 ? false : selectedTab);
```

Comment:
You should move the ternary check to a separate constant to avoid code duplication.